AstSemantics.md

```diff
@@ -66,16 +65,14 @@ WebAssembly has the following *value types*:
   * `f32`: 32-bit floating point
   * `f64`: 64-bit floating point
 
+Each parameter and local variable has exactly one value type. Function signatures
+consist of a sequence of zero or more parameter types and a sequence zero or more return
+types. (In the MVP, a function can have at most one return type).
```

Comment:
To be clear that "parameter types" and "return types" are not separate kinds, perhaps say "parameter value types" and "return value types".  Also missing "of" in "sequence zero".